app/src/datasets/chorus_dt/places_resolution.py

```diff
@@ -138,6 +130,22 @@ def resolve_place(places):
     return places, places_dict
 
 
+
+def filter_trips(trips: pd.DataFrame):
```

Comment:
I couldn't see where you call filter_trips, is it in another script ?

Reply:
It's in the main:

`    data = compute_distances(data)

    data = calc_CO2(data, carbon_dict)
    data = filter_trips(data)

    places.to_csv("/data/cleaned/places.csv", index=False)
    data.to_csv("/data/cleaned/data_chorus_dt.csv", index=True)`